### PR TITLE
Azure AI Search rebranding and messaging

### PR DESCRIPTION
**Pull request by HeidiSteen**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by HeidiSteen** [comment]
@mattmsft for visibility

File: sdk/search/Azure.Search.Documents/README.md [hunk 1]
```diff
@@ -1,46 +1,42 @@
...
 * Attach skillsets to an indexer to create searchable content from images
-  and large text documents. A skillset leverages AI from Cognitive Services
+  and unstructured documents. A skillset leverages APIs from Azure AI Services
   for built-in OCR, entity recognition, key phrase extraction, language
   detection, text translation, and sentiment analysis. You can also add
   custom skills to integrate external processing of your content during
   data ingestion.
 * In a search client application, implement query logic and user experiences
-  similar to commercial web search engines.
+  similar to commercial web search engines and copilot apps.
```
Comment (heaths) [inline sdk/search/Azure.Search.Documents/README.md:19]: Should "Copilot" be updated here, or is it not being used as a brand in this instance? I could see it either way.

**Review by heaths — APPROVED** [review]

File: sdk/search/Azure.Search.Documents/README.md (hunk 1 above)
Comment (HeidiSteen) [reply sdk/search/Azure.Search.Documents/README.md:19]: @heaths yikes, I didn't realize it was a brand (I was using the term generically).  I'll change it to "chat-style apps".

**Comment by HeidiSteen** [comment]
#sign-off

**Review by pallavit — APPROVED** [review]

**Comment by pallavit** [comment]
@HeidiSteen thank you for the PR.  @ShivangiReja please let me know if you have any concerns here. Otherwise we will go ahead and merge it on the behalf of the search team.

**Review by ShivangiReja — APPROVED** [review]

**Pull request merged**